core/arch/arm/kernel/kern.ld.S

```diff
@@ -234,6 +232,7 @@ SECTIONS
 		__data_start = .;
 		*(.data .data.* .gnu.linkonce.d.*)
 		. = ALIGN(8);
+		KEEP(*(SORT(.scattered_array*)));
```

Comment:
This will prevent the scattered arrays from being paged any longer. Another drawback is that they will become less protected at runtime since they are now mapped read-write.